sql/branch.sql

```diff
@@ -0,0 +1,3 @@
+BEGIN;
+    UPDATE payment_instructions SET due = 0 WHERE due > 9.41;
```

Comment:
> But after isolating the ones that we can't properly repair without #3851, it might not be many...

So that's just 34 people out of 654. I'm going to modify the script to set dues to zero for those 34 people (because we run the risk of overcharging them if we go with an approximation). For the rest of the users, we'll go with the original UPDATE query that sets the due to the highest multiple of the tip amount that is `<` the minimum charge